src/Sulu/Component/HttpKernel/SuluKernel.php

```diff
@@ -85,6 +85,41 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
         $this->load($loader, $confDir, '/{services}_' . $this->environment);
     }
 
+    /**
+     * The "getContainerClass" need to be normalized for preview and over contexts
+     * as its used by the symfony cache component as prefix.
+     *
+     * @see https://github.com/symfony/symfony/blob/v4.4.7/src/Symfony/Component/Cache/DependencyInjection/CachePoolPass.php#L56
+     */
+    protected function getContainerClass()
+    {
+        return $this->generateContainerClass(static::class);
+    }
+
+    /**
+     * @internal
+     *
+     * This is only for internal use. To get the container class use `getContainerClass` instead.
```

Comment:
`This is only used to support Symfony 4.4 and 5 at the same time.` - How about something like that?